media-tv/mythtv/mythtv-30.0_p20190808.ebuild

```diff
@@ -0,0 +1,369 @@
+		[ "${i}" = "native" ] && i="host"
+		myconf+=("--cpu=${i}")
+		break
+	done
+
+	# Input Support
+	myconf+=("$(use_enable input_devices_joystick joystick-menu)")
+	myconf+=("$(use_enable lirc)")
+	myconf+=("$(use_enable cec libcec)")
+	myconf+=("$(use_enable v4l v4l2)")
+	myconf+=("$(use_enable ivtv)")
+	myconf+=("$(use_enable hdpvr)")
+	myconf+=("$(use_enable ieee1394 firewire)")
+	myconf+=("$(use_enable hdhomerun)")
+	myconf+=("$(use_enable vbox)")
+	myconf+=("$(use_enable ceton)")
+	myconf+=("$(use_enable bluray libbluray_external)")
+
+	# Sound Output Support
+	myconf+=("$(use_enable oss audio-oss)")
+	myconf+=("$(use_enable alsa audio-alsa)")
+	myconf+=("$(use_enable jack audio-jack)")
+	myconf+=("$(use_enable pulseaudio audio-pulseoutput)")
+
+	myconf+=("$(use_enable systemd systemd_notify)")
+	myconf+=("$(use_enable systemd systemd_journal)")
+
+	# Bindings
+	if use perl && use python; then
+		myconf+=("--with-bindings=perl,python")
+	elif use perl; then
+		myconf+=("--without-bindings=python")
+		myconf+=("--with-bindings=perl")
+	elif use python; then
+		myconf+=("--without-bindings=perl")
+		myconf+=("--with-bindings=python")
+	else
+		myconf+=("--without-bindings=perl,python")
+	fi
+
+	use python && myconf+=("--python=${EPYTHON}")
+
+	# External codec library options (used for mythffmpeg and streaming transcode)
+	myconf+=("--enable-libmp3lame") # lame is not optional it is required for some broadcasts for silence detection of commercials
+	myconf+=("$(use_enable xvid libxvid)")
+	myconf+=("$(use_enable x264 libx264)")
+	myconf+=("$(use_enable x265 libx265)")
+	myconf+=("$(use_enable vpx libvpx)")
+
+	if use debug; then
+		myconf+=("--compile-type=debug")
+		#myconf+=("--enable-debug") does nothing per sphery
+		myconf+=("--disable-stripping") # FIXME: does not disable for all files, only for some
+		myconf+=("--enable-valgrind") # disables timeouts for valgrind memory debugging
+	else
+		myconf+=("--compile-type=release")
+	fi
+
+	# Video Output Support
+	myconf+=("--enable-x11")
+	myconf+=("--enable-xrandr")
+	myconf+=("--enable-xv")
+	myconf+=("$(use_enable vdpau)")
+	myconf+=("$(use_enable vaapi)")
+
+	# Random stuff
+	myconf+=("$(use_enable zeroconf libdns-sd)")
+	myconf+=("$(use_enable libass)")
+
+	# Clean up DSO load times and other compiler bits
+	myconf+=("--enable-symbol-visibility")
+	myconf+=("--enable-pic")
+
+	if tc-is-cross-compiler ; then
+		myconf+=("--enable-cross-compile --arch=$(tc-arch-kernel)")
+		myconf+=("--cross-prefix=${CHOST}-")
+	fi
+
+	# Build boosters
+	has distcc ${FEATURES} || myconf+=("--disable-distcc")
+	has ccache ${FEATURES} || myconf+=("--disable-ccache")
+
+	einfo "Running ./configure ${myconf[@]}"
+	./configure \
+		--cc="$(tc-getCC)" \
+		--cxx="$(tc-getCXX)" \
+		--ar="$(tc-getAR)" \
+		--extra-cflags="${CFLAGS}" \
+		--extra-cxxflags="${CXXFLAGS}" \
+		--extra-ldflags="${LDFLAGS}" \
+		--qmake=$(qt5_get_bindir)/qmake \
+		${myconf[@]}
```

Comment:
(if you disable `" "` inside array, it needs to be added around `"${myconf[@]}"`.

Reply:
So... is changing all
myconf+=("<whatever is there>")
to
myconf+=(<whatever is there>)
and using 
"${myconf[@]}"
a better style?

Reply:
I think it's less error-prone, but up to you really. It looks like this style also works.

Reply:
I'll change it. I like "less error-prone".